samples/subsys/video/capture/src/main.c

```diff
@@ -37,6 +38,23 @@ static inline int display_setup(const struct device *const display_dev)
 	       capabilities.supported_pixel_formats, capabilities.current_pixel_format,
 	       capabilities.current_orientation);
 
+	/* Set display pixel format to match the one in use by the camera */
```

Comment:
I thought about this. I think both ways are ok. The previous commit adds support for display for the current supported board RT1064 which actually works without the needs of this change because the default format of camera and display already matches. This change is needed essentially when we add support for RT1170 EVK.